join.html

```diff
@@ -171,6 +171,32 @@ <h2 class="title">Speaker Series</h2>
         </div>
     </div>
 
+    <h2 class="title">Intern Program</h2>
+    <div class="row-desktop-ratio-1-1 row-desktop-reverse">
+        <div class="col-padded v-center-children">
+            <img class="img-responsive" src="{{ site.baseurl }}/img/undraw/dev-and-teacher.svg" alt="Teacher standing in front of a blackboard and a developer coding on a laptop"/>
+        </div>
+        <div class="col-padded">
+            <p>
+                Teach LA has an awesome internship program that we'd love to have you join! Our interns focus on:
+            </p>
+                <ol>
+                    <li><b>Leading groups of developers</b> as product managers, and/or project leads to create software tools for accessible education</li>
+                    <li><b>Leading curriculum meetings </b>to create curriculum for our partnered schools</li>
+                    <li><b>Leading event planning and ideation</b>to teach at schools</li>
+                </ol>
+            <p>Here's what we do:
+            </p>
+            <ul>
+                <li>Project lead "learning labs", interactive web apps that explain CS concepts to kids</li>
+                <li>Plan events like Day of Code, a one day event where we teach students about coding!</li>
+                <li>More in depth trainings and mentorship from Teach LA dev directors in a variety of technologies</li>
+            </ul>
```

Comment:
IMO i think this is getting kinda long/messy, can we maybe just have the top one and have a link to past internship projects (which i think you are doing rn)? idk your choice tho!